addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1112,7 +1112,21 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
      * Apply the we-list on the target and rebuild the input(s)
      */
     renderListItems: async function (previewMode, value, params) {
-        const valueList = JSON.parse(value);
+        let valueList = JSON.parse(value);
+        if (this._getSelect()) {
+            // Default entry only for fields rendered as select.
+            // Remove previous default.
+            valueList = valueList.filter(value => value.id !== "" || value.display_name !== "");
```

Comment:
I'd go with:
```js
valueList = valueList.filter(value => value.id || value.display_name);
```
What do you think?

Reply:
And maybe I'd change the `valueList` name by `items` since the method name is 'renderListItems`... but we'll have a bigger diff. It's your call.

Reply:
> `valueList = valueList.filter(value => value.id || value.display_name);`

I intentionally did keep the comparison specific in order to match that specific artificial generated item - and avoid accidentally matching "falsy" cases I might not be aware of.

> `valueList` => `items`

I'd rather keep the stable commit minimal, but let's keep it in mind when adapting the commit for master.